lib/saml_idp_constants.rb

```diff
@@ -4,6 +4,7 @@ module Idp
     module Constants
       LOA1_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/1'.freeze
       LOA3_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
+      REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='.freeze
```

Comment:
Why bother with the `?ReqAttr=`?

Why not just `http://idmanagement.gov/ns/requested_attribute/`?